downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -14,7 +14,7 @@ redirect_from:
 
 ## Overview of Ballerina Swan Lake 2201.4.0
 
-<em>2201.4.0 (Swan Lake) is the fourth major release of 2022, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R5 version of the Language Specification.</em> 
+<em>2201.4.0 (Swan Lake Update 4) is the fourth major release of Ballerina Swan Lake, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R4 version of the Language Specification.</em> 
```

Comment:
Is it correct to say `fourth major release` here? Referring to the word `major` as it has a different meaning in semver. (Seems we had the same term in previous release notes too, but only noticed now)

What if we say `fourth update release` instead. @sameerajayasoma thoughts?

Reply:
`fourth update release` is the correct term.

Reply:
Updated.